src/ui/public/vis/__tests__/_agg_config.js

```diff
@@ -5,23 +5,22 @@ import { VisProvider } from 'ui/vis';
 import { AggTypesAggTypeProvider } from 'ui/agg_types/agg_type';
 import { VisAggConfigProvider } from 'ui/vis/agg_config';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
-import { RegistryFieldFormatsProvider } from 'ui/registry/field_formats';
+import { registryFieldFormats } from 'ui/registry/field_formats';
 
 describe('AggConfig', function () {
 
   let Vis;
   let AggType;
   let AggConfig;
   let indexPattern;
-  let fieldFormat;
+  const fieldFormat = registryFieldFormats;
 
   beforeEach(ngMock.module('kibana'));
   beforeEach(ngMock.inject(function (Private) {
     Vis = Private(VisProvider);
     AggType = Private(AggTypesAggTypeProvider);
     AggConfig = Private(VisAggConfigProvider);
     indexPattern = Private(FixturesStubbedLogstashIndexPatternProvider);
-    fieldFormat = Private(RegistryFieldFormatsProvider);
```

Comment:
If we move the fieldFormat registry to be like the `uiSettings` client, we'll want to keep the `fieldFormat = chrome.getFieldFormatsRegistry()` at this point so that it gets the unique registry for this test.